Game/Pregnancy/Child.gd

```diff
@@ -9,9 +10,23 @@ var fatherID: String
 var rememberedMotherName: String = "" # Required in case the character gets deleted
 var rememberedFatherName: String = ""
 var uniqueID: int
+var bornFromMonozygotic: String = ""
+const EGG_MONOZYGOTIC_LEVEL = {1: "", 2: "Twins", 3: "Triplets", 4: "Quadruplets", 5: "Quintuplets", 6: "Sextuplets"}
 
 func generateUniqueID():
 	uniqueID = GlobalRegistry.generateChildUniqueID()
+	
+func setBornFromMonozygoticStatus(monozygotic):
+	bornFromMonozygotic = EGG_MONOZYGOTIC_LEVEL[monozygotic]
+	
+func generateName():
+	match gender: "male", "peachboy":
```

Comment:
better to use `NpcGender.Male`, `NpcGender.Peachboy`. Or you can add a new static function to NpcGender like `isMasculine` that will return true for those two genders